models/adwords_connector/tmp/stg_google_ads__click_performance_tmp.sql

```diff
@@ -1,4 +1,11 @@
 {{ config(enabled=var('api_source') == 'adwords') }}
-
-select *
-from {{ var('google_ads__click_performance') }}
\ No newline at end of file
+{{
+    fivetran_utils.union_data(
+        table_identifier='click_performance',
+        database_variable='google_ads_database', 
+        schema_variable='google_ads_schema', 
+        default_database=target.database,
+        default_schema='google_ads',
+        default_variable='google_ads__click_performance'
+    )
+}}
```

Comment:
When building out the social media roll up package we found a way to integrate a custom variable for the union_schema and union_databases if this feature is used across packages in the same dbt_project. Would you be able to add the `union_schema_variable` and `union_database_variable` for these `_tmp` models.

[Here](https://github.com/fivetran/dbt_facebook_pages_source/blob/master/models/tmp/stg_facebook_pages__daily_page_metrics_total_tmp.sql#L1-L12) is an example of how we do this within the dbt_facebook_pages_source package. I would envision the variable names to be `google_ads_union_schemas` and `google_ads_union_databases` respectively.

Let me know if you have any questions on this!

Reply:
Implemented, tested locally (w/o the social media roll up package)!